database/src/adapters/assets/fungible_token_events.rs

```diff
@@ -75,7 +75,7 @@ fn compose_ft_db_events(
                         event_memo: mint_event
                             .memo
                             .clone()
-                            .unwrap_or_else(|| "".to_string())
+                            .unwrap_or_default()
```

Comment:
I looked at the docs for [`unwrap_or_default`](https://doc.rust-lang.org/std/option/enum.Option.html#method.unwrap_or_default) and now I wonder what is the default for the `memo`. I don't see a change to the structure behind `memo` with impl of a default. Did you do it earlier?

Reply:
Discussed this on the call
`.unwrap_or_default()` is the idea from clippy, it takes default value for each type
Empty string in our case